aspnetcore/blazor/file-downloads.md

```diff
@@ -52,10 +52,10 @@ For this demonstration, the app creates a 50 KB file of random data from a new b
 
 Create a `DownloadFileFromStream` method that performs the following steps:
```

Comment:
Since we're moving from reader instructions to a description of the following code ......

```suggestion
The following `DownloadFileFromStream` method performs the following steps:
```

However, more work is required later elsewhere here. I wouldn't sweat this type of thing too much at this point. All new topics receive 🦖 *The Rex Treatment*&trade; 🦖😄 at some point to clean up/improve the lingo and examples ... surface API doc cross-links ... place code into the snippet samples ... etc. Recommend making this change here but not sweating the rest for now.